get_CLN_data.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+chain=$1
+color_green=$2
+color_red=$3
+ln_git_version=$4
+
+
+echo -ne '\r### Loading CoreLN data \r'
+
+ln_running=$(systemctl is-active cln)
+coreln_color="${color_green}"
+if [ -z "${ln_running##*inactive*}" ]; then
+  ln_running="down"
+  coreln_color="${color_red}\e[7m"
+else
+  if [ -z "${ln_running##*failed*}" ]; then
+    ln_running="down"
+    coreln_color="${color_red}\e[7m"
+  else
+    ln_running="up"
+  fi
+fi
+lncli="/home/cln/lightning/cli/lightning-cli"
+
+printf "%0.s#" {1..63}
+echo -ne '\r### Loading LND data \r'
+
+
+alias_color="${color_grey}"
+ln_alias="$(${lncli} getinfo | jq -r '.alias')" 2>/dev/null
+# jq '[.duration] | add'
+ln_walletbalance="$(${lncli} listfunds | jq -r '.outputs[0].value | tonumber')" 2>/dev/null
+echo "wallet bal $ln_walletbalance"
+if [ -z "${ln_walletbalance}" ]; then
+  ln_walletbalance=0
+fi
+ln_channelbalance="$(${lncli} listfunds | jq -r '.channels[0].our_amount_msat | gsub("msat";"")| tonumber' |  awk '{print $1/1000}' )" 2>/dev/null
+echo "ln_channelbalance bal $ln_channelbalance"
+if [ -z "${ln_channelbalance}" ]; then
+  ln_channelbalance=0
+fi
+printf "%0.s#" {1..66}
+
+echo -ne '\r### Loading CoreLN data \r'
+
+ln_channels_online="$(${lncli} getinfo | jq -r '.num_active_channels')" 2>/dev/null
+ln_channels_total="$(${lncli} listincoming | jq '.[] | length')" 2>/dev/null
+node_id="$(${lncli} getinfo | jq -r '.id')" 2>/dev/null
+node_address="$(${lncli} getinfo | jq -r '.address[0].address')" 2>/dev/null
+ln_connect_addr="$node_id"@"$node_address" 2>/dev/null
+ln_connect_guidance="lightning-cli connect ${ln_connect_addr}"
+ln_external="$(echo "${ln_connect_addr}" | tr "@" " " |  awk '{ print $2 }')" 2>/dev/null
+if [ -z "${ln_external##*onion*}" ]; then
+  ln_external="Using TOR Address"
+fi
+
+printf "%0.s#" {1..70}
```

Comment:
Lines 57 - 88 seem to be LND only. Would delete them.

Reply:
I commented them out, they probably can be used later on.
https://github.com/raspibolt/raspibolt-pulse/pull/2/commits/0e11684809c1cd68d5b613fbab4e5f640304a18c